docs/source/developer/threading_implementation.rst

```diff
@@ -212,7 +212,7 @@ The general pattern for using ``get_num_threads`` in code generation is
 
    # Pass num_threads through to the appropriate backend function here
 
-See the code in ``numba/npyufunc/parfor.py``.
+See the code in ``numba/np/ufunc/parfor.py``.
```

Comment:
This is now `numba/parfors/parfor_lowering.py`

Reply:
Think this still needs an update?

Reply:
I missed this one. Thanks!